### ISSUE TITLE
Add support for more branching opcodes

### ISSUE THREAD

**Issue by x87** [sannybuilder/dev#133]
Reported by @wmysterio.

Currently the compiler only checks for opcode 004d as a condition terminator. But potentially there are more opcodes:

```
004c: jump_if_true
0AA1: return_if_false
0AA0: gosub_if_false @label
```

need a custom datatype in opcode definitions to recognize branching opcodes.

**Comment by x87** [comment]
```
{$CLEO}

0000:
while true
    if
        0AB0:   is_key_pressed 9
    004c: jump_if_true @exit
end

:exit
0A93: terminate_this_custom_script
```

throws error with Condition Check enabled.

As a workaround the conditions check must be disabled and the if command should be given a number argument:

```
{$CLEO}

0000:
while true
    if 0 // or 1,2,3...21,22,23...
        0AB0:   is_key_pressed 9
    004c: jump_if_true @exit
end

:exit
0A93: terminate_this_custom_script
```

**Comment by x87** [comment]
> A little support could be given to frame_delta_time?

Yep. See #115

**Comment by MatiDragon-YT** [comment]
@x87 what do you think of this solution?
```bash
return                ->   0051: return
return true           ->   0485: return_true
return false          ->   059A: return_false
return 0              ->   0AB2: cleo_return 0
return 1@             ->   0AB2: cleo_return 1 1@
return 3@ 5@ 7@       ->   0AB2: cleo_return 3 3@ 5@ 7@
return (4@, 6@, 8@)   ->   0AA7: function_return 4@ 6@ 8@   { I don't handle this OP }
return [5@, 7@, 9@]   ->   0AA8: method_return 5@ 7@ 9@     { I don't handle this OP }
else return           ->   0AA1: else_return
```
So as not to stray so far from the nice original syntax of SB.

**Comment by x87** [comment]
Branching information will be provided by [Sanny Builder Library](https://library.sannybuilder.com/#/) (`is_branch` attribute). As of now all modes but LCS/VCS have corresponding library definitions. For Stories https://github.com/sannybuilder/library/issues/56 needs to be implemented.

**Comment by x87** [comment]
implemented in 3.8.1
